webpack/assets/javascripts/react_app/components/common/forms/Form.stories.js

```diff
@@ -17,6 +18,8 @@ import {
   ownComponentFieldProps,
 } from './FormField.fixtures';
 
+const ControlContext = {};
```

Comment:
do we still need it?

Reply:
Yes, see line 51.
Temporary until https://github.com/theforeman/foreman/pull/7263 will be ready